src/main/java/by/itech/lab/supplier/repository/UserRepository.java

```diff
@@ -16,7 +16,7 @@
     Optional<User> findById(Long id);
 
     @Modifying
-    @Query("update User set activeForDeletion = false where id = :id")
+    @Query("update User set deleted = false where id = :id")
```

Comment:
from the logic side: if `deleted == true` - its not deleted, but if `false` - its deleted

Reply:
is it ok?

Reply:
yes it will be better

Reply:
are you going to do something with this?